apps/webpack.js

```diff
@@ -47,7 +47,7 @@ var baseConfig = {
           path.resolve(__dirname, 'test'),
           path.resolve(`${__dirname}/../dashboard/app/assets/`, 'images'),
         ],
-        loader: "url-loader?limit=1024",
+        loader: "url-loader?limit=1024&name=[name]-wp-[hash].[ext]",
```

Comment:
Even though I know it's "webpack" I can't help but read "wordpress" every time I see `-wp-` in a filename.